### ISSUE TITLE
finetuning部分代码

### ISSUE THREAD

**Issue by DopamineLcy** [pengzhiliang/MAE-pytorch#3]
非常感谢作者的工作。可以透露下finetuning部分代码什么时候会放出吗？谢谢！

**Comment by pengzhiliang** [comment]
Tomorrow or the day after tomorrow. Sorry, because I need to prepare a very important presentation(PPT) today. The accuracy can be found in
https://github.com/pengzhiliang/MAE-pytorch/issues/1#issuecomment-971079054

**Comment by pengzhiliang** [comment]
Thank u for your kind words!  If you are interested, you can try to perform pre-train first.

**Comment by DopamineLcy** [comment]
Thank you very much! I'm performing pre-training and looking forward to the fine-tuning part.
